packages/server/lib/util/specs.ts

```diff
@@ -182,32 +177,33 @@ function findSpecsOfType (searchOptions, specPattern) {
  * Resolves with an array of objects. Each object has a "testType" property
  * with one of TEST_TYPES values.
  */
-const find = (config, specPattern) => {
-  const commonSearchOptions = ['fixturesFolder', 'supportFile', 'projectRoot', 'javascripts', 'testFiles', 'ignoreTestFiles']
-
+export const find = (config: Cfg, specPattern?: string) => {
```

Comment:
While we're here, can we name this something better than `find`?

Reply:
Yes good idea!

This is still in draft BTW. I will reflag you for a review when it's ready. I think we can rejig the logic a bit; right now there are a lot of "hacks" for CT.